WordPress/gutenberg
# Update Welcome Guides to use external image URLs

## Description

Updates the welcome guides in `edit-post`, `edit-widgets` and `customize-widgets` so that they use images that are served from `wp-admin/images/block-editor/*` instead of `data:` URLs that are included in the JavaScript bundle. This should significantly reduce the bundle sizes of `edit-post`, `edit-widgets` and `customize-widgets.`

https://github.com/WordPress/wordpress-develop/pull/1269 is the Core PR which adds the images to `wp-admin/images/block-editor`.

## How has this been tested?

You'll be testing https://github.com/WordPress/wordpress-develop/pull/1269 and this PR at the same time.

1. Clone the `wordpress-develop` repo, check out https://github.com/WordPress/wordpress-develop/pull/1269, build using `npm i; grunt build`.
2. Clone the `gutenberg` repo, check out https://github.com/WordPress/gutenberg/pull/32026, build using `npm i; npm run build`.
3. Configure your environment to run both local copies. For example, if using `wp-env`, modify `gutenberg/.wp-env.json` to set `"core": "/path/to/wordpress-develop/build"`, then run `wp-env start`.
4. Go to Posts → Add New, open the Options menu, and click Welcome Guide. The guide's images should appear normally.
5. Go to Appearance → Widgets, open the Options menu, and click Welcome Guide. The guide's images should appear normally.
6. Go to Appearance → Customize → Widgets, open the Options menu, and click Welcome Guide. The guide's image should appear normally.

## Checklist:
